crates/fluvio-sc/src/services/public_api/topic/create.rs

```diff
@@ -85,7 +85,7 @@ async fn validate_topic_request(name: &str, topic_spec: &TopicSpec, metadata: &C
         return Status::new(
             name.to_string(),
             ErrorCode::TopicInvalidName,
-            Some(format!("Invalid topic name: '{}'. Topic name can contain only lowercase alphanumeric characters or '-'.", name.to_string())),
```

Comment:
Clippy fix is in: https://github.com/infinyon/fluvio/pull/2109, so might remove it from here.